packages/framework/src/Facades/Features.php

```diff
@@ -40,6 +40,7 @@ public static function enabled(string $feature): bool
             static::bladePages(),
             static::markdownPages(),
             static::documentationPages(),
+            static::publications(),
```

Comment:
This feature method is not needed as publications are in a package. To enable/disable the feature install/uninstall the package.

Reply:
Resolved in https://github.com/hydephp/develop/commit/40d6ee367d8778842d6cc1e359043fcb7b848770